client/src/includes/panels.ts

```diff
@@ -78,6 +135,25 @@ export function initCollapsiblePanels(
   toggles.forEach(initCollapsiblePanel);
 }
 
+/**
+ * Initialises functionality to expand and contract all panels at once
+ */
+export function initCollapseAllPanels(
+  button = document.querySelector<HTMLButtonElement>(
+    '[data-all-panels-toggle]',
+  ),
```

Comment:
I note this is also collapsing the `title` section where this button is invisible. Is that intentional?